deploy/7-deploy-pendle-oracle.ts

```diff
@@ -4,49 +4,119 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { ADDRESSES } from "../helpers/deploymentConfig";
 
+type OracleConfig = {
+  name: string;
+  market: string;
+  token: string;
+  collateral: string;
```

Comment:
I would use the term `underlyingToken`, as we are using in the PendleOracle contract